serviceuser.go

```diff
@@ -473,15 +481,17 @@ func (su *ServiceUser) CGet(qrLevel QRLevel, filter []*dicom.Element,
 		doassert(event.command != nil)
 		resp, ok := event.command.(*dimse.CGetRsp)
 		if !ok {
-			return fmt.Errorf("Found wrong response for C-GET: %v", event.command)
+			return fmt.Errorf("found wrong response for C-GET: %v", event.command)
 		}
-		if resp.Status.Status != dimse.StatusPending {
-			if resp.Status.Status != 0 {
-				e := fmt.Errorf("Received C-GET error: %+v", resp)
-				dicomlog.Vprintf(0, "dicom.serviceUser: C-GET: %v", e)
-				return e
-			}
-			break
+		switch resp.Status.Status {
+		case dimse.StatusPending:
+			continue receiveLoop
```

Comment:
for consistent sake, could you add the same logging statement everywhere as in CFind handler above?

Reply:
Ok